scripts/start-registry-and-push-images.fish

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env fish
```

Comment:
do we really need this script? most systems have bash already so the shebang line would cover handling your other script

Reply:
I can delete it!